### PR TITLE
Wrong URLs

### PR DESCRIPTION
**Pull request by KPostOffice**
## Description
Grab correct url from webhook

### CODE REVIEW

**Comment by fridex** [comment]
It looks to me like a fix for https://github.com/thoth-station/user-api/issues/524

File: thoth/user_api/api_v1.py [hunk 1]
```diff
@@ -535,10 +535,10 @@ def schedule_kebechet(body: dict):
     headers = connexion.request.headers
     if "X-GitHub-Event" in headers:
         service = "github"
-        url = body["repository"]["url"]
+        url = body["repository"]["html-url"]
```
Comment (goern) [inline thoth/user_api/api_v1.py]: @harshad16 is it html-url or html_url ?

Comment (harshad16) [reply thoth/user_api/api_v1.py]: yes it is html_url as per https://developer.github.com/v3/activity/events/types/#webhook-payload-example-34
my bad, missed it while reviewing.

Comment (goern) [reply thoth/user_api/api_v1.py]: not your bad!

**Pull request merged**